designsafe/apps/api/messages/tests.py

```diff
@@ -0,0 +1,39 @@
+from designsafe.apps.api.messages.models import PageAlert
+from designsafe.apps.auth.views import load_alerts_into_session, agave_oauth_callback
+from django.test import TestCase
+from django.test.client import RequestFactory
+from django.forms.models import model_to_dict
+from django.contrib.sessions.middleware import SessionMiddleware
+from django.contrib.messages.storage.fallback import FallbackStorage
+from mock import patch, Mock
+
+alerts = []
```

Comment:
Not ideal to use a model level constant. This can be done on `setUp` or `setUpTestData` and attach it to the class.

Reply:
Fixed.